knowledge-base/inputs-handle-keyboard-events.md

```diff
@@ -32,13 +32,15 @@ The keyboard events bubble up the DOM, so you can attach event handlers for them
 
 If you are looking for an event when the user confirms an action, consider the built-in `OnChange` event which fires when the user presses `Enter` or blurs the input.
 
+>important Optionally, you can set the [**`DebounceDelay`**]({%slug components/textbox/overview%}#features) parameter of the textbox to 0. The `DebounceDelay` default value is 150. It can fire the `onkeypress` event logic before the [**`ValueChanged`**]({%slug components/textbox/events%}#valuechanged) event of the input. 
```

Comment:
Maybe we should explain this even better - does the DebounceDelay value determine the order of events? What is "it" in "it can fire"? DebounceDelay surely does not fire `keypress`.